### PR TITLE
Adds network information to be collected as part of diagnostic bundle

### PR DESCRIPTION
**Pull request by GoelDeepak**
## High-level description

This patch adds few network related information to be collected as part of diagnostics bundle. 

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS_OSS-3613](https://jira.mesosphere.com/browse/DCOS_OSS-3613) Improve diagnostics bundle to include information that would help debugging network issues

## Checklist for all PRs

  - [ ] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change:
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]
___
**PLEASE FILL IN THE TEMPLATE ABOVE** / **DO NOT REMOVE ANY SECTIONS ABOVE THIS LINE**

## Instructions and review process

**What is the review process and when will my changes land?**

All PRs require 2 approvals using GitHub's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).

Reviewers should be:
* Developers who understand the code being modified.
* Developers responsible for code that interacts with or depends on the code being modified.

It is best to proactively ask for 2 reviews by @mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the [next train](https://github.com/dcos/dcos/blob/master/contributing.md).

### CODE REVIEW

**Comment by mesosphere-mergebot-dev** [comment]
This repo has @mesosphere-mergebot integration. You can interact with the following commands.
```
@mesosphere-mergebot bump-ee  
@mesosphere-mergebot merge-it  
@mesosphere-mergebot override-status  
@mesosphere-mergebot label [Ship It |Holding |Work In Progress |Ready For Review |Request For Comment] 
```

* PR creators can apply one of  `[Ready For Review |Work In Progress]`. [Owners](https://github.com/dcos/dcos/blob/master/owners.json) can apply any label.

**Comment by GoelDeepak** [comment]
@mesosphere-mergebot bump-ee

**Comment by GoelDeepak** [comment]
@mesosphere-mergebot bump-ee

**Comment by GoelDeepak** [comment]
@mesosphere-mergebot bump-ee

**Comment by GoelDeepak** [comment]
@urbanserj @vespian Please review

**Comment by GoelDeepak** [comment]
@mesosphere-mergebot label Ready For Review

File: gen/dcos-config.yaml [hunk 1]
```diff
@@ -830,11 +840,32 @@ package:
...
                   "Command": ["dmesg"]
+              },
+              {
+                  "Command": ["ip", "addr"]
+              },
+              {
+                  "Command": ["ip", "route"]
+              },
+              {
+                  "Command": ["ifconfig", "-a"]
```
[the remark this answers is not in the document]
Comment (GoelDeepak) [reply gen/dcos-config.yaml]: Unfortunately, anything that requires `root` privilege cannot be fetched as dcos-diagnostics runs without root privilege. That would take out all `dcos-net-env` commands as well as `iptables-save`. `uname -a` information is already present in dmesg. I have added collection of vips.

[the remark this answers is not in the document]
Comment (urbanserj) [reply gen/dcos-config.yaml]: I see, That's such a bad limitation.

**Comment by GoelDeepak** [comment]
@mesosphere-mergebot bump-ee

**Comment by urbanserj** [comment]
@mesosphere-mergebot override-status teamcity/dcos/test/azure/arm DCOS_OSS-2115

**Comment by GoelDeepak** [comment]
@mesosphere-mergebot bump-ee

**Comment by mesosphere-mergebot-dev** [comment]
:wave: This PR has been inactive for 3 days. Please review the status checks to see what needs to be done to move it forward or change the label to Work in Progress. Thank you!

**Comment by urbanserj** [comment]
@mesosphere-mergebot label [Ship It]

File: gen/dcos-config.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (gpaul) [reply gen/dcos-config.yaml]: Weird - not having root permissions to load stuff into the diagnostics bundle makes no sense.

[the remark this answers is not in the document]
Comment (gpaul) [reply gen/dcos-config.yaml]: https://jira.mesosphere.com/browse/DCOS_OSS-3877

**Pull request merged**